pandora-common/src/assets/definitions.ts

```diff
@@ -110,6 +111,12 @@ export interface AssetBaseDefinition<Type extends AssetType, A extends AssetDefi
 	 *  - `undefined` temporarily until all previews are added
 	 */
 	preview?: string | null;
+
+	/**
+	 * Override the default asset preference
+	 *  - only 'maybe' is allowed since 'favorite', 'normal', 'prevent' and 'doNotRender' does not make sense
```

Comment:
Why does 'prevent' make no sense?

I feel we would want to set the default of password locks to "prevent", not to 'maybe'.

Reply:
wouldn't maybe be enough, I mean you would get prompted, but sure I allow prevent as well

Reply:
added prevent as an option too